
ArtifactUsage usage
uint32 id
